generated/csharp/GitHub/Middleware/Options/VersionOptions.cs

```diff
@@ -0,0 +1,13 @@
+using Microsoft.Kiota.Abstractions;
+
+namespace GitHub.Octokit.Client.Middleware.Options;
+
+public class VersionOptions : IRequestOption
+{
+
+    public string APIVersion { get; set; } = GetAPIVersion();
+
+  private static string GetAPIVersion() =>
+    //TODO : Get the version from the generator
```

Comment:
I'm curious how this would be possible. Perhaps we could (before/after generating) save it somewhere from the YAML spec if there's not an explicit artifact containing it in the generated code.

Reply:
Yeah the idea here is to pull this out into a config when we generate from source so that it always matches what it's generated from. I have not done that yet (obviously) ;)

Reply:
:+1: I'm happy to merge as-is and iterate down the road.